### PR TITLE
Use Fastlane/Danger for CI

### PR DESCRIPTION
**Pull request by huminzhi**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/forcedotcom/SalesforceMobileSDK-iOS.svg?branch=unstable)](https://travis-ci.org/forcedotcom/SalesforceMobileSDK-iOS)
+[![Build Status](https://gist.githubusercontent.com/huminzhi/74e0b81003314709ded9c23386ec28ba/raw/status.png)](https://mobilesdk.ci.data.com/job/MobileSDK-iOS)
```
Comment (bhariharan) [inline readme.md]: Is this link externally accessible? Otherwise, this won't work.

**Comment by huminzhi** [comment]
@bhariharan I revert back the readme as we need to discuss further, but the other changes are still valid. Please review. Thanks!

File: fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
+lane :CI do
```
Comment (bhariharan) [inline fastlane/Fastfile]: @huminzhi Will this work with `Travis CI` too, if we decide to go with that?

File: DangerFile [hunk 3]
```diff
@@ -0,0 +1,9 @@
+junit.parse "test_output/report.junit"
```
Comment (bhariharan) [inline DangerFile]: @huminzhi Will this work with `Travis CI` too, if we decide to go with that?

Comment (huminzhi) [inline DangerFile]: Yes, danger/fastlane are working with Travis CI also..

**Pull request merged**